<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="WKWZTPqvmdckJHX" failures="2" tests="3" name="com.qa.centrum.eng.test.CentrumSampleCouponsTest" time="28.267" errors="0" timestamp="2022-10-11T14:51:57 IST" skipped="0">
  <testcase classname="com.qa.centrum.eng.test.CentrumSampleCouponsTest" name="cookieButton" time="12.914"/>
  <system-out/>
  <testcase classname="com.qa.centrum.eng.test.CentrumSampleCouponsTest" name="verifyRecommendedArticleOne" time="9.103">
    <failure message="expected [content/cf-consumer-healthcare/bp-wellness-centrum/en_US/home/learn/articles/nutrients-and-routine/the-abc-s-of-vitamin-d.html] but found [learn/articles/nutrients-and-routine/the-abc-s-of-vitamin-d/]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [content/cf-consumer-healthcare/bp-wellness-centrum/en_US/home/learn/articles/nutrients-and-routine/the-abc-s-of-vitamin-d.html] but found [learn/articles/nutrients-and-routine/the-abc-s-of-vitamin-d/]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.baseClass.BaseClass.compareTwoText(BaseClass.java:739)
at com.qa.centrum.eng.test.CentrumSampleCouponsTest.verifyRecommendedArticleOne(CentrumSampleCouponsTest.java:67)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyRecommendedArticleOne -->
  <system-out/>
  <testcase classname="com.qa.centrum.eng.test.CentrumSampleCouponsTest" name="verifyRecommendedArticleTwo" time="6.250">
    <failure message="expected [content/cf-consumer-healthcare/bp-wellness-centrum/en_US/home/learn/articles/nutrients-and-routine/how-to-take-vitamins-when.html] but found [learn/articles/nutrients-and-routine/how-to-take-vitamins-when/]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [content/cf-consumer-healthcare/bp-wellness-centrum/en_US/home/learn/articles/nutrients-and-routine/how-to-take-vitamins-when.html] but found [learn/articles/nutrients-and-routine/how-to-take-vitamins-when/]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.baseClass.BaseClass.compareTwoText(BaseClass.java:739)
at com.qa.centrum.eng.test.CentrumSampleCouponsTest.verifyRecommendedArticleTwo(CentrumSampleCouponsTest.java:79)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyRecommendedArticleTwo -->
  <system-out/>
</testsuite> <!-- com.qa.centrum.eng.test.CentrumSampleCouponsTest -->
